### ISSUE TITLE
Error: undefined identifier AT_SYMLINK_NOFOLLOW in Alpine Linux (musl libc)

### ISSUE THREAD

**Issue by LAC-Tech** [vibe-d/vibe.d#2540]
Hello, I'm guessing this an issue with musl systems. Not sure if it's fixable but here's how to reproduce it if you're interested

```
LAPTOP-162NUQO7:~$ dub init testvibed -t vibe.d
Package recipe format (sdl/json) [json]:
Name [testvibed]:
Description [A simple vibe.d server application.]:
Author name [lewis]:
License [proprietary]:
Copyright string [Copyright © 2021, lewis]:
Add dependency (leave empty to skip) []:
Successfully created an empty project in '/home/lewis/testvibed'.
Package successfully created in testvibed
LAPTOP-162NUQO7:~$ cd testvibed/
LAPTOP-162NUQO7:~/testvibed$ dub
Running pre-generate commands for vibe-d:tls...
Performing "debug" build using /usr/bin/dmd for x86_64.
taggedalgebraic 0.11.19: target for configuration "library" is up to date.
eventcore 0.9.13: target for configuration "epoll" is up to date.
stdx-allocator 2.77.5: target for configuration "library" is up to date.
vibe-core 1.13.0: building configuration "epoll"...
../.dub/packages/vibe-core-1.13.0/vibe-core/source/vibe/core/file.d(1046,46): Error: undefined identifier AT_SYMLINK_NOFOLLOW
../.dub/packages/vibe-core-1.13.0/vibe-core/source/vibe/core/file.d(1053,60): Error: no property st_mtimensec for type core.sys.posix.sys.stat.stat_t
../.dub/packages/vibe-core-1.13.0/vibe-core/source/vibe/core/file.d(1054,59): Error: no property st_ctimensec for type core.sys.posix.sys.stat.stat_t
/usr/bin/dmd failed with exit code 1.
```

Happy to provide more info/testing if needed.

**Comment by Geod24** [comment]
Sorry for the late reply. This was a missing upstream definition: https://github.com/dlang/druntime/pull/3333
It has been fixed in v2.096.0 (and the compiler on edge has been updated), so it shouldn't be a problem anymore with DMD.
In addition, there's a workaround in vibe-core to prevent this from happening again: https://github.com/vibe-d/vibe-core/commit/69444da9a3637635ecd0b647b8ff6a8c37b81470

However, I see that the workaround hasn't been released yet. Will fix.

**Comment by Geod24** [comment]
Done: https://github.com/vibe-d/vibe-core/releases/tag/v1.14.0
